src/headless/converse-muc.js

```diff
@@ -973,7 +973,7 @@ converse.plugins.add('converse-muc', {
             getAllKnownNicknamesRegex () {
                 const longNickString = this.getAllKnownNicknames().join('|');
                 const escapedLongNickString = p.escapeRegexString(longNickString)
-                return RegExp(`(?:\\s|^)@(${escapedLongNickString})(?![\\w@-])`, 'ig');
+                return RegExp(`(?:\\s|^).?@(${escapedLongNickString})(?![\\w@-])`, 'ig');
```

Comment:
Instead of matching `.?` shouldn't it be matching `opening_mention_characters`?

Reply:
Definitely. Used that as a test and forgot to update afterwards. Thanks!